documentapi/src/tests/policies/policies_test.cpp

```diff
@@ -83,10 +83,10 @@ class Test : public vespalib::TestApp {
     void requireThatExternPolicyWithUnknownPatternSelectsNone();
     void requireThatExternPolicySelectsFromExternSlobrok();
     void requireThatExternPolicyMergesOneReplyAsProtocol();
-    void requireThatStoragePolicyWithIllegalParamIsAnErrorPolicy();
-    void requireThatStoragePolicyIsRandomWithoutState();
-    void requireThatStoragePolicyIsTargetedWithState();
-    void requireThatStoragePolicyCombinesSystemAndSlobrokState();
+    void requireThatContentPolicyWithIllegalParamIsAnErrorPolicy();
+    void requireThatContentPolicyIsRandomWithoutState();
+    void requireThatContentPolicyIsTargetedWithState();
+    void requireThatContentPolicyCombinesSystemAndSlobrokState();
```

Comment:
Should we retain at least one test per language that tests that `Storage` still works?

Reply:
Yes.

Reply:
I think there weren't any tests that "Content" worked, before ... 🙀

Reply:
So in Java-land there is no test of this setup — only a mock setup where some random string (`"storage"`) is used as the policy name. 
In C++ land there is a test which involves getting a policy from a semi-real mbus. The simplest thing to do would probably just be to verify that whatever policy was returned for `"Storage"` was the same as what was returned for `"Content"` — do you feel the call?

Reply:
It's really a trivial thing, though. Not absolutely convinced we need it.